components/plurals/src/lib.rs

```diff
@@ -95,6 +97,7 @@ use rules::runtime::test_rule;
 /// A type of a plural rule which can be associated with the [`PluralRules`] struct.
 ///
 #[derive(Debug, PartialEq, Eq, Clone, Copy, Hash)]
+#[allow(clippy::exhaustive_enums)] // this type is stable
```

Comment:
Nit: This looks like the kind of thing that we may want to extend in the future, if CLDR adds more types of plural forms.